src/routes/docs/products/functions/domains/+page.markdoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: article
+title: Domains
+description: Execute Appwrite Functions through domains using standard HTTP GET, POST, or other request methods to serve static, JSON, HTML, or other content.
+---
+
+Each deployed function can have its own domain, generated or developer defined.
+You can use this domain to execute Appwrite Functions through HTTP methods. 
+You can use common practices like using paths, query parameters, headers, HTTP methods, formdata, 
+and all the typical HTTP concepts to implement Appwrite Functions.
+
+{% arrow_link href="/docs/products/functions/develop" %}
+Learn about Function development
+{% /arrow_link %}
+
+# Generated domains {% #generated-domains %}
+
+1. In the Appwrite Console's sidebar, click **Functions**.
```

Comment:
These steps are all 1.

Reply:
Yes markdown renders them as 1.2.3.4 ;) This in fact works